src/scheduler/data/mod.rs

```diff
@@ -20,3 +20,5 @@ pub trait DataSource: Debug {
     /// Generate the next non-deterministic `u64` value to return to a requesting task.
     fn next_u64(&mut self) -> u64;
 }
+
+pub use random::RandomDataSource;
```

Comment:
nit: put this right underneath `pub(crate) mod random;`

Reply:
Done